prompting/miners/langchain_miner.py

```diff
@@ -0,0 +1,169 @@
+# The MIT License (MIT)
```

Comment:
Why we're adding a deprecated miner to the codebase (that wasn't there before)? I'm sure there's a good reason, just not clear to me

Reply:
This is also some old code on main, and probably this miner should be deleted